alphadia/transferlearning/train.py

```diff
@@ -32,7 +32,8 @@
     # --------- USer settings ------------
     "batch_size": 1000,
     "max_lr": 0.0005,
-    "train_ratio": 0.8,
+    "train_ratio": 0.7,
```

Comment:
I know its not required for calculation but let's add also test_ratio. What do you think about calling it `_fraction` instead of ratio? Is it really a rati